packages/core/contracts/insured-bridge/BridgeAdmin.sol

```diff
@@ -219,23 +219,28 @@ contract BridgeAdmin is BridgeAdminInterface, Ownable, Lockable {
      * domain messenger.
      * @dev msg.value must equal to l1CallValue.
      * @param chainId L2 network ID where Deposit contract is deployed.
-     * @param l2Token address of L2 token to enable/disable deposits for.
+     * @param l1Token address of L1 Token to enable/disable deposits and relays for.
      * @param depositsEnabled bool to set if the deposit box should accept/reject deposits.
      * @param l1CallValue Amount of ETH to include in msg.value. Used to pay for L2 fees, but its exact usage varies
      * depending on the L2 network that this contract sends a message to.
      * @param l2Gas Gas limit to set for relayed message on L2.
      * @param l2GasPrice Gas price bid to set for relayed message on L2.
      * @param maxSubmissionCost: Arbitrum only: fee deducted from L2 sender's balance to pay for L2 gas.
      */
-    function setEnableDeposits(
+    function setEnableDepositsAndRelays(
```

Comment:
this function now takes in the `l1Token` instead of the `l2Token`. this enables us to fetch (and call) the associated `bridgePool` from within the `_whitelistedTokens` mapping.